.github/workflows/check.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
-          node-version: "16"
+          node-version: "18"
```

Comment:
Why need node 18?

Reply:
I don't know. Just copied from the clickhouse PR😂.

Reply:
Try to keep node 16 and wait the CI 😄

Reply:
Hmm, I try to change it back, and it also failed in my fork repo. https://github.com/washanhanzi/metabase-databend-driver/actions/runs/6515055462/job/17697018583

Reply:
I don't know what is `build-static-viz` doing. With version 18 the fork repo action failed so I just comment it and build the driver.

Reply:
Hmm, the upstream change it https://github.com/metabase/metabase/commit/dbd4fbaac7a6c8fd626e7604d5dee47a59e27647

Reply:
Oh I got it. Just merge this pr first and I will adjust the CI yaml. Thanks!

Reply:
Ok, thanks.